null_safety_examples/layout/base/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: layout
+description: >
+  Sample app from "Building Layouts", https://flutter.io/docs/development/ui/layout.
```

Comment:
flutter.io?  That's an old URL.. Also, relative link is best.

Reply:
Done.